test/common/testlib.py

```diff
@@ -1171,12 +1169,12 @@ def assert_pixels(self, selector: str, key: str,
             for layout in self.layouts:
                 if layout["name"] not in skip_layouts:
                     self.set_layout(layout["name"])
+                    time.sleep(wait_delay)
                     if "rtl" in self.current_layout["name"]:
                         self._set_direction("rtl")
                     self.assert_pixels_in_current_layout(selector, key, ignore=ignore,
                                                          scroll_into_view=scroll_into_view,
-                                                         wait_animations=wait_animations,
-                                                         wait_delay=wait_delay)
+                                                         wait_animations=wait_animations)
 
                     if "rtl" in self.current_layout["name"]:
                         self._set_direction("ltr")
```

Comment:
What if we sleep here? As this is the issue right? We switch the layout back but it takes longer then expected.

Reply:
Hmm that's not the issue. The failing pixel test, is supposed to be RTL but it's showing the original UI. So the problem is not when resetting the direction to ltr.

Reply:
Aha! But we sleep now before we switch rtl? So is the issue that something is still rendering before we switch rtl?

Reply:
That's exactly my assumption